chrome/views/popupView.html

```diff
@@ -9,54 +9,57 @@
         <a id="close-button" title="Close">&times;</a>
     </div>
     <div class="popupMsg">Loading...</div>
-    <div class="content" style="display: none;">
-        <div class="gttTopRight">
-            <div style="float: left;" class="clearfix listrow">
-                <ul id="gttPosition" />
+    <div class="outerContent">
+        <div id="threadCards" style="display: none;"></div>
```

Comment:
This is the only new div on the page. It shows the threadCards at the top of the popup.